src/types/monitoring.ts

```diff
@@ -0,0 +1,337 @@
+import { PublicKey } from '@solana/web3.js';
```

Comment:
We need a readme about monitoring to document it please. It seems very generic and i'm not sure we will use Slack, Discord, mail for example